modules/assim.sequential/R/downscale_function.R

```diff
@@ -82,28 +119,30 @@ SDA_downscale_preprocess <- function(data_path, coords_path, date, carbon_pool)
   n <- length(y)
   indices <- seq_len(n)
   folds <- split(indices, cut(seq_len(n), breaks = k, labels = FALSE))
-  
+
   if (!returnTrain) {
-    folds <- folds  # Test indices are already what we want
+    folds <- folds # Test indices are already what we want
   } else {
-    folds <- lapply(folds, function(x) indices[-x])  # Return training indices
+    folds <- lapply(folds, function(x) indices[-x]) # Return training indices
   }
-  
+
   if (!list) {
     folds <- unlist(folds)
   }
-  
+
   return(folds)
 }
 
+
+
 ##' @title SDA Downscale Function
 ##' @name SDA_downscale
 ##' @author Joshua Ploshay, Sambhav Dixit
 ##'
 ##' @param preprocessed List. Preprocessed data returned as an output from the SDA_downscale_preprocess function.
-##' @param date Date. If SDA site run, format is yyyy/mm/dd; if NEON, yyyy-mm-dd. Restricted to years within file supplied to 'preprocessed' from the 'data_path'.
-##' @param carbon_pool Character. Carbon pool of interest. Name must match carbon pool name found within file supplied to 'preprocessed' from the 'data_path'.
-##' @param covariates SpatRaster stack. Used as predictors in downscaling. Layers within stack should be named. Recommended that this stack be generated using 'covariates' instructions in assim.sequential/inst folder
+##' @param date Date. If SDA site run, format is yyyy/mm/dd; if NEON, yyyy-mm-dd. Restricted to years within file supplied to 'preprocessed' from the 'ensemble_data'.
```

Comment:
same comment about fixing date

Reply:
In this case, it doesn't look like the function actually uses date, so I've removed it